pkg/cmd/triggertemplate/describe.go

```diff
@@ -174,13 +184,10 @@ func printTriggerTemplateDescription(s *cli.Stream, p cli.Params, ttname string)
 	return w.Flush()
 }
 
-func getResourceTemplate(triggerTemplate v1alpha1.TriggerTemplate) string {
-	var out bytes.Buffer
-	resourceTemplate, err := json.Marshal(triggerTemplate.Spec.ResourceTemplates)
+func getResourceTemplate(resourceTemplate v1alpha1.TriggerResourceTemplate) *unstructured.Unstructured {
+	d, err := runtime.DefaultUnstructuredConverter.ToUnstructured(&resourceTemplate)
 	if err != nil {
-		log.Printf("json marshal failed with error: %v", err)
-		return ""
+		return nil
```

Comment:
if error, we should provide the error to the user.

Reply:
Handled error part example testcase can find [here](https://github.com/tektoncd/cli/pull/977/files?file-filters%5B%5D=.golden#diff-0b974d78d9d0e10479b541532fff804dR9)